metadata.themoviedb.org.python/HOW_TO_RELEASE.md

```diff
@@ -0,0 +1,6 @@
+1. Write changelog in changelog.txt
```

Comment:
Can you drop this file from the submission? If using GitHub actions please add it to gitattributes as export-ignore

Reply:
done